scripts/ci/lib.sh

```diff
@@ -299,6 +335,11 @@ push_matching_collector_scanner_images() {
         fi
     }
 
+    if [[ "$arch" != "amd64" ]]; then
+        echo "Skipping rebundling for non-amd64 arch"
```

Comment:
Collector images are available for power, but we will need to add arm

Reply:
Yeah will need to fix this, if it's alright, I'd like to fix it in a later PR

Reply:
sounds good